iotedgedev/cli.py

```diff
@@ -122,10 +121,17 @@ def e2e(ctx):
                   short_help="Add a new module to the solution",
                   help="Add a new module to the solution, where NAME is the module name")
 @add_module_options(envvars)
+@click.option("--file",
+              "-f",
+              "template_file",
+              default=envvars.DEPLOYMENT_CONFIG_TEMPLATE_FILE,
+              show_default=True,
+              required=False,
+              help="Specify the deployment manifest template file")
 @with_telemetry
-def add(name, template, group_id):
+def add(name, template, group_id, template_file):
```

Comment:
will the module be added to debug template also?

Reply:
Yes, in https://github.com/Azure/iotedgedev/pull/358/files#diff-81a619e7b82430890d648131b2db0338R111

Reply:
Or make it an override and default to the env var setting.

Reply:
@jongio It already defaults to env var setting (https://github.com/Azure/iotedgedev/pull/358/files#diff-f7f95b5a237aedfa9c80a3ec57d525e7R127). I am working on updating it to default to `envvars.DEPLOYMENT_CONFIG_FILE_PATH`, so that users can specify a built deployment manifest.